lib/Doctrine/Migrations/Tools/Console/Command/ExecuteCommand.php

```diff
@@ -67,7 +68,7 @@ protected function configure() : void
                 'Time all the queries individually.'
             )
             ->setHelp(<<<EOT
-The <info>%command.name%</info> command executes a single migration version up or down manually:
+The <info>%command.name%</info> command executes migration versions up or down manually:
 
     <info>%command.full_name% YYYYMMDDHHMMSS</info>
```

Comment:
This help test should include an example how to run multiple migrations. In addition to that, we should use a FQCN for the migrations instead of the legacy date format.

Reply:
done